ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -192,6 +134,13 @@ nnfw_open (const GstTensorFilterProperties * prop, void **private_data)
     }
   }
 
+  /* validate model file first */
+  if (!g_file_test (prop->model_files[0], G_FILE_TEST_IS_REGULAR)) {
```

Comment:
ONE runtime should receive the model path. The path can be a regular file or a directory.

```suggestion
  if (!g_file_test (prop->model_files[0], (G_FILE_TEST_EXISTS | G_FILE_TEST_IS_DIR))) {
```

Reply:
I added new commit to handle directory string in tensor-filter. Please review the changes again.